utilities/pipelines/dependencies/Microsoft.Compute/galleries/parameters/parameters.json

```diff
@@ -5,6 +5,11 @@
         "name": {
             "value": "adp<<namePrefix>>azsigweux001"
         },
+        "locks": {
```

Comment:
I remember we went back and forth many times on deciding if applying locks on dependencies or not. Do we agree to add them now? I'm not necessarily against it, just curious

Reply:
My bad. This was not intentional. I will remove them again.

Reply:
Removed